draft-packages/avatar/KaizenDraft/Avatar/decision-tokens.scss

```diff
@@ -1,16 +1,10 @@
 @import "~@kaizen/design-tokens/sass/color";
 @mixin avatar-theme-switching-tokens() {
-  --avatar--color-icon-color: rgba(var(#{$color-purple-800-rgb-id}), 0.7);
+  --avatar--color-icon-color: rgba($color-purple-800-rgb, 0.7);
 }
-$dt-color-background-color: var($color-gray-300-id, $color-gray-100);
-$dt-color-background-color-initials: var($color-gray-300-id, $color-purple-200);
-$dt-color-background-color-personal: var(
-  $color-orange-100-id,
-  $color-orange-200
-);
+$dt-color-background-color: $color-gray-300;
+$dt-color-background-color-initials: $color-gray-300;
+$dt-color-background-color-personal: $color-orange-100;
 $dt-color-text-color: $color-purple-800;
 $dt-color-border-color: $color-white;
-$dt-color-icon-color: var(
-  --avatar--color-icon-color,
-  rgba($color-purple-800-rgb, 0.3)
-);
+$dt-color-icon-color: var(--avatar--color-icon-color);
```

Comment:
I'm a little bit iffy about the mixin + CSS var `--avatar--color-icon-color` here: does this need to be a CSS var or can we change `$dt-color-icon-color: var(--avatar--color-icon-color);` to `$dt-color-icon-color: rgba($color-purple-800-rgb, 0.7);` ?

This is the only place the mixin is being used: 
![image](https://user-images.githubusercontent.com/7651503/133951793-e2273326-9208-4f8f-9d12-7d3c286deaac.png)

Reply:
Its hard to gauge the intended behaviour, because it's totally reasonable to think that the fallback `rgba($color-purple-800-rgb, 0.3)` was intentionally used whenever the mixin `avatar-theme-switching-tokens` isn't.

Reply:
Got rid of the mixin :)